sycl/test-e2e/DeviceImageBackendContent/CUDA_interop_test.cpp

```diff
@@ -0,0 +1,33 @@
+// RUN: %{build} %cuda_options -o %t.out
+// RUN: %{run} %t.out
+// REQUIRES: cuda, cuda_dev_kit
+
+#include <cuda.h>
+#include <sycl/backend.hpp>
+#include <sycl/detail/core.hpp>
+#include <vector>
+
+int main() {
+  sycl::device d([](const sycl::device &d) {
+    return d.get_backend() == sycl::backend::ext_oneapi_cuda;
```

Comment:
Isn't this guaranteed by the requirements? I would expect a default-created device to pick up a CUDA device.